Android/app/src/androidTest/java/com/example/kahye/myapplication/ExampleInstrumentedTest.java

```diff
@@ -0,0 +1,26 @@
+package com.example.kahye.myapplication;
+
+import android.content.Context;
+import android.support.test.InstrumentationRegistry;
+import android.support.test.runner.AndroidJUnit4;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+import static org.junit.Assert.*;
+
+/**
+ * Instrumented test, which will execute on an Android device.
+ *
+ * @see <a href="http://d.android.com/tools/testing">Testing documentation</a>
+ */
+@RunWith(AndroidJUnit4.class)
+public class ExampleInstrumentedTest {
+    @Test
+    public void useAppContext() {
+        // Context of the app under test.
+        Context appContext = InstrumentationRegistry.getTargetContext();
+
+        assertEquals("com.example.kahye.myapplication", appContext.getPackageName());
```

Comment:
Is this a test app? Otherwise, we should make a proper package name rather than com.example.kahye.myapplication